dot_tool-versions.tmpl

```diff
@@ -0,0 +1,7 @@
+{{- if .neovim_nightly_native_lsp }}
```

Comment:
Have to get asdf to install my default node version as I need some default packages installing. Depending on which version of node neovim uses I could get around this if I can figure out a way of globally install npm packages with nix's node
https://github.com/tapayne88/dotfiles/pull/5/files#r583052487